kernel_test.go

```diff
@@ -15,6 +16,249 @@ const (
 	success = "\u2713"
 )
 
+const (
+	connectionKey = "a0436f6c-1916-498b-8eb9-e81ab9368e84"
+	sessionID     = "ba65a05c-106a-4799-9a94-7f5631bbe216"
+	transport     = "tcp"
```

Comment:
Is the transport likely to change?  Maybe we can inline that, as I doubt that we will be running with other types of transport?